packages/myst-to-typst/src/container.ts

```diff
@@ -104,6 +104,11 @@ export const containerHandler: Handler = (node, state) => {
     return;
   }
 
+  if (node.enumerator?.endsWith('.1')) {
```

Comment:
There should be a better way to do this... But it's not simple, since `myst-to-typst` is taking a fully-processed tree, it's lost some of the intermediate knowledge of _how_ things are counted, and just has the final results.

Doing something like stashing the entire `numbering` object on each enumerated node could work, but that feels yucky. Maybe there is a happy medium?

But for now, I think the existing shortcut in this PR is ok. It handles default enumerators.

Reply:
Can you add a comment to this line with that?